ocrd_tesserocr/common.py

```diff
@@ -66,12 +172,18 @@ def image_from_page(workspace, page,
                  'y': 0,
                  'w': page_image.width,
                  'h': page_image.height}
+    # FIXME: uncomment as soon as we get @orientation in PageType:
+    # # region angle: PAGE orientation is defined clockwise,
+    # # whereas PIL/ndimage rotation is in mathematical direction:
+    # page_xywh['angle'] = -(page.get_orientation() or 0)
     # FIXME: remove PrintSpace here as soon as GT abides by the PAGE standard:
```

Comment:
@tboenig What is the status of the `Border` vs. `PrintSpace` issue?

Reply:
https://github.com/OCR-D/assets/issues/34

> We will replace PrintSpace with Border and in the spec (ocr-d.github.io/gt//trans_documentation/lySatzspiegel.html) we add information for using Border.

@tboenig Let me know if you need help with automating the conversion.